.gitignore

```diff
@@ -1,4 +1,2 @@
 /venv
-.idea
```

Comment:
Why remove this?

Reply:
In an older commit i saw the folder was removed so i assumed it wasn't needed anymore

Reply:
No, the problem is it's for editors. Jetbrains IDEs use it for storing their information about a project. Like .vscode, so it's there to prevent people from pushing their .vscode and or .idea.

Reply:
ahh alright

Reply:
i put it back again

Reply:
Other then that, great PR man. Thank you!